software/docmost.yml

```diff
@@ -0,0 +1,11 @@
+name: "docmost"
+website_url: "https://docmost.com/"
+source_code_url: "https://github.com/docmost/docmost"
+description: "Docmost is an open-source collaborative wiki and documentation software. It is an open-source alternative to Confluence and Notion."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Nodejs
+tags:
+  - Knowledge Management Tools
```

Comment:
I am not sure, if this tag is needed as it seems primarily to be a wiki (and therefore the wiki tag should be enough already) but let's wait for some other reviewers on that

Reply:
Seeing as knowledge management tools tag is related to wikis, yeah this is redundant.

Reply:
done